src/pages/home/Home.jsx

```diff
@@ -1,9 +1,8 @@
 import React, { useEffect, useState } from "react";
 import "./home.css";
-import ArticleCard from "../../components/ArticleCard/articleCard/ArticleCard";
+import ArticleCard from "../../components/article_card/ArticleCard";
```

Comment:
This should still be in PascalCase in my opinion, you're referring to a component.

Reply:
not sure why I thought all folders are in lower when I did that, will amend